cdap-kafka/src/main/java/co/cask/cdap/kafka/run/KafkaServerMain.java

```diff
@@ -55,10 +57,11 @@ public static void main(String [] args) throws Exception {
   public void init(String[] args) {
     CConfiguration cConf = CConfiguration.create();
     String zkConnectStr = cConf.get(Constants.Zookeeper.QUORUM);
-    String zkNamespace = cConf.get(KafkaConstants.ConfigKeys.ZOOKEEPER_NAMESPACE_CONFIG);
+    String zkNamespace = cConf.get(KafkaConstants.ConfigKeys.ZOOKEEPER_NAMESPACE_CONFIG_DEPRECATED);
+
+    int port = cConf.getInt(KafkaConstants.ConfigKeys.PORT_CONFIG_DEPRECATED, -1);
```

Comment:
I don't think we need to get the port. It will be set as part of the `generateKafkaConfig`